platform/jewel/markdown/core/src/main/kotlin/org/jetbrains/jewel/markdown/rendering/DefaultInlineMarkdownRenderer.kt

```diff
@@ -168,7 +168,21 @@ public open class DefaultInlineMarkdownRenderer(rendererExtensions: List<Markdow
         currentTextStyle: TextStyle,
     ) {
         // Each image source corresponds to one rendered image.
-        appendInlineContent(node.source, "![${node.title}](${node.source})")
+        appendInlineContent(
+            node.source,
+            buildString {
+                append(" ![")
```

Comment:
Is this extra space in the start intended?

Reply:
Yes otherwise when you select and copy to the clipboard there is no space around this. It's a workaround :)